pkg/updatestrategy/rolling_update_test.go

```diff
@@ -129,11 +129,11 @@ func getFailureDomain(nodes []*Node) string {
 		}
 	}
 
-	min := len(nodes)
+	minNode := len(nodes)
```

Comment:
Just a nit, can we refactor `minNode` to `minNodes`, just to make it clear it holds the min node count, and not some index.